tasks/main.yml

```diff
@@ -3,35 +3,120 @@
 
 - include_vars: "{{ ansible_os_family }}.yml"
 
+- assert:
+    msg: dpb_cvsroot must not be empty
+    that:
+      - dpb_cvsroot is defined
+      - dpb_cvsroot | length > 0
+
+- name: Get release version
+  command: uname -r
+  register: register_release_version
+  changed_when: false
+
+- set_fact:
+    dpb_release: "{{ register_release_version.stdout }}" # x.y
```

Comment:
i am bit reluctant to use `ansible_distribution_release` because the definition of the value is not clear. it has been broken in certain versions of `ansible`. in some platforms, it is `x.y`, in others, `x.y-RELEASE`, or `x.y-p1`. the commands, taken from the official documentation, used in the task will never break.